docs/src/man/bootc-edit.md

```diff
@@ -4,8 +4,8 @@ bootc-edit - Apply full changes to the host specification
 
 # SYNOPSIS
 
-**bootc-edit** \[**-f**\|**\--filename**\] \[**\--quiet**\]
-\[**-h**\|**\--help**\] \[**-V**\|**\--version**\]
+**bootc edit** \[**-f**\|**\--filename**\] \[**\--quiet**\]
```

Comment:
These ones are currently generated by `cargo xtask manpages` using https://docs.rs/mandown/latest/mandown/

So any fixes would need to land in that code.

Reply:
OK, thank you @cgwalters. If so, I will remove changes for manpage, and make this PR only for mdbook. For the naming issue, I will research docgen, and try to fix it in a separate PR.